src/control/jobs/control_jobs.c

```diff
@@ -940,7 +940,7 @@ static int32_t dt_control_delete_images_job_run(dt_job_t *job)
           do
           {
             char *xmp_filename = g_utf16_to_utf8(data.cFileName, -1, NULL, NULL, NULL);
-            files = g_list_append(files, g_build_filename(dirname, xmp_filename));
+            files = g_list_append(files, g_build_filename(dirname, xmp_filename, NULL));
```

Comment:
Uh, why did this work in the first place?
Are there more occurrences?

Reply:
clang gave me a proper error message here (sorry I have not mentioned it above):
`error: missing sentinel in function call [-Werror,-Wsentinel]
            files = g_list_append(files, g_build_filename(dirname, xmp_filename));`

This is the only instance I have found. Interesting that travis' clang did not captured that.